pkg/node/node.go

```diff
@@ -111,24 +115,38 @@ func (s *Service) Init() error {
 		var initiators []string
 
 		switch arr.BlockProtocol {
+		case common.NVMETransport:
+			if len(nvmeInitiators) == 0 {
+				return fmt.Errorf("NVMe transport was requested but NVMe initiator is not available")
+			}
+			s.useISCSI = false
+			s.useFC = false
 		case common.ISCSITransport:
 			if len(iscsiInitiators) == 0 {
 				return fmt.Errorf("iSCSI transport was requested but iSCSI initiator is not available")
 			}
+			s.useISCSI = true
 			s.useFC = false
 		case common.FcTransport:
 			if len(fcInitiators) == 0 {
 				return fmt.Errorf("FC transport was requested but FC initiator is not available")
 			}
 			s.useFC = true
+			s.useISCSI = false
 		default:
+			s.useISCSI = len(iscsiInitiators) > 0
```

Comment:
incase both iscsi and fc initiators are present, useISCSI and useFC will be set to true here. Is it expected?